addons/project/tests/test_multicompany.py

```diff
@@ -454,12 +436,10 @@ def test_cross_subtask_project(self):
                 with self.debug_mode():
                     with Form(self.env['project.task'].with_context({'tracking_disable': True})) as task_form:
                         task_form.name = 'Test Subtask in company B'
+                        task_form.project_id = self.task_1.project_id
                         task_form.parent_id = self.task_1
-
                     task = task_form.save()
 
-                self.assertEqual(task.project_id, task.parent_id.project_id, "The subtask should have the same project as its parents")
-                self.assertEqual(task.company_id, task.parent_id.company_id, "The company of the subtask should be the one from its parent.")
```

Comment:
It seems we have now a limitation, no? 🤔 

Or the behavior changed?

I am wondering if we should not add an onchange to automatically set the project of the parent task if the project_id was unset.

What do you think?